python/pyiceberg/expressions/base.py

```diff
@@ -19,58 +19,22 @@
 from abc import ABC, abstractmethod
 from dataclasses import dataclass
 from functools import reduce, singledispatch
-from typing import Generic, TypeVar
+from typing import (
+    ClassVar,
+    Generic,
+    Literal,
```

Comment:
I think we want to import our own literal from:
```python
from pyiceberg.expressions.literals import Literal
```
instead of the one from typing.

Reply:
Ah, I was wondering what the problem was. Thanks!